Formula/dwarfutils.rb

```diff
@@ -1,5 +1,5 @@
 class Dwarfutils < Formula
-  desc "lib and utility to dump and produce DWARF debug info in ELF objects"
+  desc "Lib and utility to dump and produce DWARF debug info in ELF objects"
```

Comment:
I seem to remember trying that in the first place, but it hit the "80 characters" rule, and I didn't know how to shorten it coherently.